mqtt_locator.py

```diff
@@ -54,17 +60,25 @@ def __init__(self, config):
         pygame.mouse.set_visible(False)
         self.pings = []
         self.config = config
+
         self.client = mqtt.Client()
         self.client.on_connect = self.on_connect
         self.client.on_message = self.on_message
         self.client.connect(self.config["host"],
                             int(self.config["port"]),
                             int(self.config["keepalive"]))
-        self.background = pygame.image.load("map.PNG")
-        self.x_shift = self.background.get_width() / 2.0
-        self.y_shift = self.background.get_height() / 2.0
-        self.x_scale = self.x_shift / 180.0
-        self.y_scale = self.y_shift /  90.0
+
+        self.background = pygame.image.load(self.config['map_image'])
+
+        self.proj_in = pyproj.Proj(proj='latlong', datum='WGS84')
```

Comment:
I feel like as long as there is a library that supports arbitrary image projections the projection and data-formats should be config variables like the map.